assets/xml/objects/object_oF1d_map.xml

```diff
@@ -1,108 +1,142 @@
 ﻿<Root>
+    <!-- Object file for Gorons -->
     <File Name="object_oF1d_map" Segment="6">
-        <DList Name="object_oF1d_map_DL_0003D0" Offset="0x3D0" />
-        <DList Name="object_oF1d_map_DL_000458" Offset="0x458" />
-        <DList Name="object_oF1d_map_DL_0008C0" Offset="0x8C0" />
-        <DList Name="object_oF1d_map_DL_000948" Offset="0x948" />
-        <DList Name="object_oF1d_map_DL_000D50" Offset="0xD50" />
-        <DList Name="object_oF1d_map_DL_000DD8" Offset="0xDD8" />
-        <DList Name="object_oF1d_map_DL_001560" Offset="0x1560" />
+        <!-- Snowball -->
+        <!-- Large Snowball Fragment -->
+        <DList Name="gGoronLargeSnowballFragmentMaterialDL" Offset="0x3D0" />
+        <DList Name="gGoronLargeSnowballFragmentModelDL" Offset="0x458" />
+
+        <!-- Medium Snowball Fragment -->
+        <DList Name="gGoronMediumSnowballFragmentMaterialDL" Offset="0x8C0" />
+        <DList Name="gGoronMediumSnowballFragmentModelDL" Offset="0x948" />
+
+        <!-- Small Snowball Fragment -->
+        <DList Name="gGoronSmallSnowballFragmentMaterialDL" Offset="0xD50" />
+        <DList Name="gGoronSmallSnowballFragmentModelDL" Offset="0xDD8" />
+
+        <DList Name="gGoronSnowballDL" Offset="0x1560" />
         <Blob Name="object_oF1d_map_Blob_001750" Size="0x20" Offset="0x1750" />
-        <Texture Name="object_oF1d_map_Tex_001770" OutName="tex_001770" Format="rgba16" Width="32" Height="64" Offset="0x1770" />
+        <Texture Name="gGoronSnowBallTex" OutName="goron_snow" Format="rgba16" Width="32" Height="64" Offset="0x1770" />
         <Blob Name="object_oF1d_map_Blob_002770" Size="0x800" Offset="0x2770" />
-        <Texture Name="object_oF1d_map_Tex_002F70" OutName="tex_002F70" Format="i4" Width="32" Height="32" Offset="0x2F70" />
-        <DList Name="object_oF1d_map_DL_0031A0" Offset="0x31A0" />
-        <DList Name="object_oF1d_map_DL_003258" Offset="0x3258" />
+
+        <!-- Steam -->
+        <Texture Name="gGoronSteamTex" OutName="goron_steam" Format="i4" Width="32" Height="32" Offset="0x2F70" />
+        <DList Name="gGoronSteamMaterialDL" Offset="0x31A0" />
+        <DList Name="gGoronSteamModelDL" Offset="0x3258" />
+
+        <!-- Unused Effects -->
         <DList Name="object_oF1d_map_DL_003280" Offset="0x3280" />
```

Comment:
I don't think it really matters what we call them if they're unused anyway, most of the time it's not reasonable to speculate on the use.